migrate-to-github/README.md

```diff
@@ -0,0 +1,77 @@
+  - only when Github, Gitlab API doesn't allow something (e.g. `git reset`). In other cases Ogr
+    includes all the stuff we need.
+- That's it
+  
+### What we need to use to achieve the goals in [what-to-migrate.md]()
+
+- Create a bot account which will transfer the PRs/Issues to Github
+- Create Github mirror of Copr project
+  - Copr migrated from Github to Pagure, where it had already some 30 PRs. The project
+    on Pagure did not migrate these PRs and started on PR id 2 -> the previous 30 PRs
+    are lost from context. But in the current Github mirror these PRs are there and
+    we need to create a new clean mirror to match all the Pagure IDs.
+- Give this bot rights to write to Copr project
+- Create Github API token for this bot account
+  - Security: Give the token minimal amount of privileges -> it won't need to delete
+    something so don't enable this! (but on the other hand if something will be
+    screwed up - it's still just a new mirror repo so no big deal :D) And delete
+    the token once migration is done.
+  - The API token needs to read and write, check these options when creating API 
+    (I am still checking if we really needs all of these):
+  - https://docs.github.com/en/developers/apps/building-oauth-apps/scopes-for-oauth-apps#available-scopes
+    - public_repo
+    - read:project
+    - read:user (for migrating assignees)
+- Create Pagure API token
+  - Security: the same reason as above
+  - The API token needs only to read the project, nothing else.
+  - Documentation missing???:
+    - ???
+- Create a config for the script
+- Run the script
+- Have a cup of coffee and relax
+
+#### What is done and what needs to be done (some kind of progressbar about the script):
+
+As I mentioned at the meeting, when I was sick, I wrote a lot of code that has nothing to do
+with our needs. In the future, when I'm sick again, I plan to finish the script. However, for
+now I'm just concentrating on finishing the parts of the script that are relevant to us. Below 
+I have listed the points that concern us and are/aren't finished. After discussing which things
+we want to convert and which things we don't, I'll edit this list and focus only on these points.
+
+- [x] Logic of transferring between git forges
+- [x] Some king of parser which will decide how to parse data to another forge
+- [ ] Transfer issues logic:
+  - [ ] Transfer comments (should be easy using the Ogr API) - 1/2h work
+  - [x] Everything else
+- [ ] Transfer PRs logic
+  - [x] Asynchronously create branches for making a PR
+  - [x] Transfer PR's body and diff
+  - [ ] Transfer PR's comment - 1/2h work
+    - transferring the content is easy, but some comments may not be relevant (after rebase
+      they may react to a line which is not valid to the comment anymore) 
+- [x] Match ids
+- [ ] Migrate labels - should be easy to do - Ogr provides API (1/2h work)
+- [ ] Tests - this one will take some time
+- [ ] Be actually able to run the script
+
```

Comment:
We IMHO want to add some additional information to every PR/issue/comment

Let's take the pagure issue

```
#2355 copr-ping nagios check seems broken 
Opened 3 days ago by praiskup. Modified 3 days ago

I disabled 'copr-backend.target' more than an hour ago, and still no email about copr-ping task failure.... we should take a look what is going on.
```

Once migrated to GitHub, the issue will have the same title and text, but I think at the end it should say something like

```
---
Original issue: https://pagure.io/copr/copr/issue/2355
Opened: 2022-10-29
Opened by: @praiskup
```

Or something like this. We should figure out the format and location in some dedicated PR for your migration script.